### ISSUE TITLE
Glitch / blanking while hecking for uncomitted changes

### ISSUE THREAD

**Issue by device-host** [Murmele/Gittyup#214]

**Comment by device-host** [comment]
The app is unusable for me - building release from source (I also tried the latest GitAhead version and have exact same issue).
I'm running on Linux (debian bullseye based).

The center pane goes blank (white) for about 2 seconds and left shows the loading animation and 'checking for uncommitted changes'. It is doing this so often that the app is unusable (and may be a risk for epileptics from all the flashing!).

I really like the app otherwise - maybe a temporary 'pause update' could be added to the UI next to the update button? Or an option to adjust the auto update time (if it is not event based?). My repo is really small but maybe that would be generally useful for others too with larger repos where updates might be slower / cpu intensive.

**Comment by Murmele** [comment]
Hi @device-host it should not update every few seconds, only if you have local changes. So that is not the usual behavior.

Which version of flatpak do you have on linux?

**Comment by device-host** [comment]
Just to be clear the flatpak version doesn't work at all for me (the UI doesn't respond to clicks) so I built from source and the issue described here occurs on release build (and old versions like GitAhead). I mention flatpak because I tried it as a solution - I have the 'old-stable' Flatpak 1.10.7 from the debian repo.

I did some further investigation and found that If i create a repo on another drive the issue stops (no 'checking for uncommitted changes' loop so could my issue be some software triggering a file event every few seconds? I've confirmed no files are actually changing so could it be fired by some other software also scanning the files for changes?

**Comment by Murmele** [comment]
> I did some further investigation and found that If i create a repo on another drive the issue stops (no 'checking for uncommitted changes' loop) so could my issue be some other software scanning the same files and triggering a file event that disturbs GittyUp every few seconds? I've confirmed no files are actually changing so I'm trying to see what's happening with commands like 'lsof +D'

Sounds like, but it is strange that it changes the file so that the indexer triggers a new change

**Comment by Murmele** [comment]
I don't think we will implement something here, because in normal cases it makes no sense to stop indexing because then you might get confused why it does not update. I think the best you ask in a Xilinx forum what is it and if it can be turned off.
